rules/windows/create_stream_hash/create_stream_hash_susp_ip_domains.yml

```diff
@@ -0,0 +1,38 @@
+title: Unusual File Download from Direct IP Address
+id: 025bd229-fd1f-4fdb-97ab-20006e1a5368
+status: experimental
+description: Detects the download of suspicious file type from URLs with IP
+author: Nasreddine Bencherchali
+references:
+    - https://www.ultimatewindowssecurity.com/securitylog/encyclopedia/event.aspx?eventid=90015
+date: 2022/09/07
+logsource:
+    product: windows
+    category: create_stream_hash
+detection:
+    selection_domain:
+        Contents|contains:
```

Comment:
why not `://1` ...
should have less FP , or not :smile:

Reply:
Yeah, i think it's better than what i did :D